code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm

```diff
@@ -537,23 +590,437 @@
+		You won't always have that sort of time however, this is what the Conversion ritual is for. By making an unbeliever appear before Nar-Sie, their eyes will open \
+		in a matter of seconds, that is, if their mind can handle it. Those either too weak, or of an impenetrable mind will be purged, and devoured by Nar-Sie. \
+		In this case, their remains will be converted into a container where to retrieve their belongings, along with a portion of their blood. \
+		Also, know that you can quicken the ritual by wearing formal cult attire, and that the vessel will remain incapacitated for the duration of the ritual."
+	var/remaining = 100
+	var/mob/living/carbon/victim = null
+	var/flavor_text = 0
+	var/success = 0
+	var/list/impede_medium = list(
+		"Security Officer",
+		"Warden",
+		"Detective",
+		"Head of Security",
+		"Internal Affairs Agent",
+		"Head of Personnel",
+		)
+	var/list/impede_hard = list(
+		"Chaplain",
+		"Captain",
+		)
+	var/obj/effect/cult_ritual/conversion/conversion = null
+
+
+/datum/rune_spell/conversion/Destroy()
+	if(conversion)
+		conversion.Die()
+	..()
+
+/datum/rune_spell/conversion/update_progbar()//progbar tracks conversion progress instead of paid blood
+	if (!progbar)
+		progbar = image("icon" = 'icons/effects/doafter_icon.dmi', "loc" = spell_holder, "icon_state" = "prog_bar_0")
+		progbar.pixel_z = WORLD_ICON_SIZE
+		progbar.plane = HUD_PLANE
+		progbar.layer = HUD_ABOVE_ITEM_LAYER
+		progbar.appearance_flags = RESET_COLOR
+	progbar.icon_state = "prog_bar_[min(100,round((100-remaining), 10))]"
+	return
+
+/datum/rune_spell/conversion/cast()
+	var/obj/effect/rune/R = spell_holder
+	R.one_pulse()
+	var/turf/T = R.loc
+	var/list/targets = list()
+
+	//first lets check for a victim above
+	for (var/mob/living/carbon/C in T)//all carbons can be converted...but only carbons. no cult silicons.
+		if (!iscultist(C))
+			//TODO: MOB NEEDS A MIND, leaving as is for now, so I can convert dummies to test stuff
+			targets.Add(C)
+	if (targets.len > 0)
+		victim = pick(targets)
+	else
+		to_chat(activator, "<span class='warning'>There needs to be a potential convert standing or lying on top of the rune.</span>")
+		qdel(src)
+		return
+
+	update_progbar()
+	if (activator.client)
+		activator.client.images |= progbar
+
+	//secondly, let's stun our victim and begin the ritual
+	to_chat(victim, "<span class='danger'>Occult energies surge from below your feet and seep into your body.</span>")
+	victim.Silent(5)
+	victim.Knockdown(5)
+	victim.Stun(5)
+	victim.overlay_fullscreen("conversionborder", /obj/abstract/screen/fullscreen/conversion_border)
+	victim.overlay_fullscreen("conversionred", /obj/abstract/screen/fullscreen/conversion_red)
+	victim.update_fullscreen_alpha("conversionred", 255, 5)
+	victim.update_fullscreen_alpha("conversionborder", 255, 5)
+	conversion = new(T)
+	flick("rune_convert_start",conversion)
+	playsound(R, 'sound/effects/convert_start.ogg', 75, 0, -4)
+
+	if (victim.mind)
```

Comment:
You don't seem to have actually increased the remaining time in the event the person is in impede_medium, impede_hard, or implanted, despite this reading as it being the case.

Reply:
The value of "remaining" isn't increased, it's the progress done every second that is reduced. In other words, if you were to remove the implant through surgery mid conversion, then the rest of the conversion would progress much faster.